package.json

```diff
@@ -50,31 +50,31 @@
     "deepmerge": "^4.2.2",
     "element-closest": "^3.0.0",
     "escape-string-regexp": "^4.0.0",
-    "firebase": "^7.23.0",
+    "firebase": "^7.24.0",
     "intl": "^1.2.5",
     "jstimezonedetect": "^1.0.7",
     "leaflet": "^1.7.1",
     "lodash.debounce": "^4.0.8",
     "loglevel": "^1.7.0",
-    "markdown-it": "^11.0.1",
-    "markdown-it-emoji": "^1.4.0",
+    "markdown-it": "^12.0.0",
```

Comment:
need to check breaking changes

Reply:
Looks ok... https://github.com/markdown-it/markdown-it/blob/master/CHANGELOG.md#1200---2020-10-14